.changeset/warm-poems-swim.md

```diff
@@ -0,0 +1,5 @@
+---
+"@fuel-ts/signer": minor
+---
+
+Stopped exporting secp256k1 curve
```

Comment:
Isn't this off?

```suggestion
Replacing `elliptic` with `@noble/curves`
```

Reply:
No, because [here](https://github.com/FuelLabs/fuels-ts/pull/1601/files#diff-76473a2f7b3391ac4ca49a7c64262b5520fa1eccc77fd003589fe3a23348d9bfL18) I stopped exporting the `getCurve` method. Replacing `elliptic` with `@noble/curves` was a purely internal change that I don't think should be mentioned in the changeset because nothing changed which the end user of `@fuel-ts/signer` would notice (besides `getCurve` not being exported anymore which I mentioned).

I stopped exporting `getCurve` because exporting the full secp256k1 curve from any package isn't the job of `@fuel-ts/signer`. We're just the consumers of an implementation of the curve. If somebody else wants this same curve, they themselves can put `elliptic`, `@noble/curves` or something third in their dependencies and use it as they wish.

Reply:
Can we use multiline text in these changelogs?

```
 - Replacing `elliptic` with `@noble/curves`
 - Stopped exporting `getCurve()` / secp256k1
```

Trivial details, but something we definitely [need](https://github.com/FuelLabs/fuels-ts/issues/1028) to [improve](https://github.com/FuelLabs/fuels-ts/issues/970).

Reply:
Sure, I can do that. Conceptually, I was guided by the idea that changesets should contain only the descriptions for changes that are observable to the end user.

Reply:
I hope the formatting is not affected when rendering the final consolidated changelog.

We will see. :)